src/Tickets/Commerce/Admin/Orders_Page.php

```diff
@@ -180,6 +201,38 @@ public function is_admin_orders_page() {
 		return true;
 	}
 
+	/**
+	 * Checks if the current screen is a single admin order page.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_admin_single_page() {
```

Comment:
I feel like we should have a utility to do a lot of this. Just a thought... this feels like it is doing something commonly needed but tailored for a specific use. 

Just my 2 cents..

Reply:
@stratease i feel you and i think you are very right - In terms of this project unfortunately i can't implement it within it but it definitely makes sense to have something like this. Probably in an inno day or another phase of OM if there is time :D